src/domain/safe/entities/schemas/multisig-transaction.schema.ts

```diff
@@ -1,6 +1,60 @@
 import { Schema } from 'ajv';
 import { buildPageSchema } from '@/domain/entities/schemas/page.schema.factory';
+import { buildZodPageSchema } from '@/domain/entities/schemas/page.schema.factory';
+import { z } from 'zod';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { HexSchema } from '@/validation/entities/schemas/hex.schema';
+import { NumericStringSchema } from '@/validation/entities/schemas/numeric-string.schema';
+import { DataDecodedSchema } from '@/domain/data-decoder/entities/schemas/data-decoded.schema';
+import { Operation } from '@/domain/safe/entities/operation.entity';
+import { SignatureType } from '@/domain/messages/entities/message-confirmation.entity';
 
+export const ConfirmationSchema = z.object({
+  owner: AddressSchema,
+  submissionDate: z.coerce.date(),
+  transactionHash: HexSchema.nullish().default(null),
+  signatureType: z.nativeEnum(SignatureType),
+  signature: HexSchema.nullish().default(null),
+});
+
+export const MultisigTransactionSchema = z.object({
```

Comment:
The `multisigTransactionPageSchema` and `MULTISIG_TRANSACTION_PAGE_SCHEMA_ID` are no longer needed right?